command/auth_enable.go

```diff
@@ -307,6 +306,10 @@ func (c *AuthEnableCommand) Run(args []string) int {
 		if fl.Name == flagNameTokenType {
 			authOpts.Config.TokenType = c.flagTokenType
 		}
+
+		if fl.Name == "plugin-version" {
```

Comment:
extract this out to a variable, `flagNamePluginVersion`, maybe, similar to `flagNameTokenType`?

Reply:
Yeah good point 👍 it's duplicated a lot now.